src/global_launch/config/README.md

```diff
@@ -250,7 +250,7 @@ specified within an array: one for the `x` component, and one for the `y` compon
 `wind_sensor.generator_type` is `gaussian`.
 - _Datatype_: `double` array, length 2
 - _Range_: `(0.0, MAX_DOUBLE)`
-    - If a standard deviation of zero is desired, then consider using the constant generator instead.
+  - If a standard deviation of zero is desired, then consider using the constant generator instead.
```

Comment:
I believe this change here is causing the markdown lint check to fail with the follow error:
```
Error: src/global_launch/config/README.md:253:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
```